src/site/content/en/blog/oyo-case-study/index.md

```diff
@@ -0,0 +1,182 @@
+Until recently, the OYO team was offering both a
+[Progressive Web App (PWA)](https://developers.google.com/web/progressive-web-apps)
+and a native Android app to achieve that goal.
+The Android app had significantly higher engagement:
+users converted three times as often as those using the PWA.
+But users also tended to uninstall the Android app over time
+because of concerns about storage space.
+
+To reduce the Android app's footprint on users' devices
+while keeping the benefits of the native experience,
+the team decided to turn to
+[Trusted Web Activities (TWAs)](https://developers.google.com/web/updates/2019/02/using-twa).
+
+## What's a TWA?
+Before Chrome 72, Android developers who wanted to display web content in
+their native apps had to use
+[WebView](https://developer.android.com/reference/android/webkit/WebView),
+which came with some meaningful limitations:
+it's not as fast as Chrome and doesn't include all of Chrome's APIs and features.
+So, if you wanted behavior that WebView's rendering engine didn't support,
+you had to build your own browser around it—which isn't exactly trivial!
+
+[Trusted Web Activities (TWAs)](https://developers.google.com/web/updates/2019/02/using-twa)
+address those limitations by showing web content directly in Chrome.
+Breaking down the TWA name helps explain its features:
+- An _activity_ is one screen or view in an Android app's user interface.
+- TWAs use Chrome to display _web_ content for the app's activities.
+- A TWA's content is _trusted_ because it uses
+  [Digital Asset Links](https://developers.google.com/digital-asset-links/v1/getting-started)
+  to verify that the same person created the Android app
+  and the web content that it's displaying.
+
+{% Aside %}
+It's easy to confuse PWAs and TWAs.
+_PWAs_ use web technologies to create experiences comparable to native apps.
+_TWAs_ allow you to display a PWA in an Android app wrapper
+that can be downloaded from the Google Play Store.
+{% endAside %}
+
+## Why create a TWA rather than a native app?
+
+More and more Android apps are delivering content from developers' own websites.
+TWAs acknowledge that reality by offering the best
+of the native and web app worlds:
+- They have all the expected functionality of native Android apps,
+  including a launcher icon, push notifications, and fullscreen display.
+- They offer the performance and features of Chrome.
+- They use the version of Chrome installed on the device,
+  so they always have the latest APIs and features.
+- They use significantly less storage than a fully native app,
+  which is a concern for many users,
+  especially those with lower-end devices.
+
+Running in Chrome also has a number of handy perks.
+For example, TWAs use the data in Chrome's cache,
+including cookies and anything stored using the
+[Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API).
+One benefit of this setup is that users stay logged in
+across the browser and the TWA app.
```

Comment:
> For example, TWAs use the data in Chrome's cache, including cookies and anything stored using the [Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API).

Is this the source of the idea? https://youtu.be/6lHBw3F4cWs?t=443

I'm not familiar with TWAs (and am assuming that you haven't done much with them either, but please correct me if I'm wrong) so I'm trying to gauge whether we should get a technical review of the content from someone on the TWA team.

The reason I got confused is because list item 4 in the introduction of [this post](https://developers.google.com/web/updates/2019/02/using-twa) seems to say something to the contrary (but it's jargon-y so I might just be misunderstanding):

> The host app doesn't have direct access to web content in a Trusted Web activity or any other kind of web state, like cookies and localStorage. Nevertheless, you can coordinate with the web content by passing data to and from the page in URLs (e.g. through query parameters, custom HTTP headers, and intent URIs.)

Reply:
I noticed that discrepancy also, but the Medium post made this claim, so I left it in, assuming they knew what they were doing. I'd definitely be in favor of a technical review.

Reply:
[This YouTube video](https://youtu.be/TCgT8dzSiU8) confirms that the statement in the post is correct.